shm/Cargo.cpp

```diff
@@ -1,11 +1,19 @@
-#include <Cargo.hpp>
+#include "Cargo.hpp"
 
 Cargo& Cargo::operator+=(const size_t amount) {
-    m_amount += amount;
+    _amount += amount;
     return *this;
 }
 
 Cargo& Cargo::operator-=(const size_t amount) {
-    m_amount -= amount;
+    _amount -= amount;
     return *this;
 }
+
+bool Cargo::operator==(const Cargo& src) const {
+     return (_name == src._name) && (_amount == src._amount) && (_basePrice == src._basePrice);
+}
+
+bool Cargo::operator!=(const Cargo& src) const {
+    return !operator==(src);

```

Comment:
Do we need both operators (== and !=)?
I think that operator == is enough.

Reply:
I know that only one operator should be added in the task. In my opinion these operators are symmetrical, so I added both. Similarly to the rule of zero/three/five. Some programmers also add operators of relation "<" and ">".

Reply:
OK, I don't know those rules yet.